packages/synapse-react-client/src/synapse-queries/user/useFavorites.ts

```diff
@@ -85,20 +84,45 @@ export function useGetFavorites(
     UseQueryOptions<PaginatedResults<EntityHeader>, SynapseClientError>
   >,
 ) {
-  const { accessToken, keyFactory } = useSynapseContext()
-
-  return useQuery({
-    ...options,
-    queryKey: keyFactory.getUserFavoritesQueryKey(sort, sortDirection),
-    queryFn: () =>
-      SynapseClient.getUserFavorites(
```

Comment:
In SynapseClient, we also have a `getAllOfPaginatedService` which may be preferable to implementing this in a hook